server/app/featureflags/FeatureFlags.java

```diff
@@ -36,6 +36,11 @@ public final class FeatureFlags {
 
   private final Config config;
 
+  // Address correction and verifcation flags
+  private static final String ESRI_ADDRESS_CORRECTION_ENABLED = "esri_address_correction_enabled";
+  private static final String ESRI_ADDRESS_SERVICE_AREA_VALIDATION_ENABLED =
+      "esri_address_verification_enabled";
```

Comment:
Non-blocking: Should this be "esri_address_service_area_validation_enabled" to match the value in the application.conf?

Reply:
yes it should, thank you for catching it!

Reply:
updated to "esri_address_service_area_validation_enabled"